docs/operation/cluster_and_service_configuration.rst

```diff
@@ -68,20 +73,17 @@ To add a new static user, perform the following operations:
                       userID: "<uuidv4>"
       [...]
 
-#. Save and apply the changes.
+#. Save the ConfigMap changes.
 
-#. From the Bootstrap node, execute the following command which connects to
-   the Salt master container and applies salt-states to propagate the new
-   changes down to the underlying services.
-
-   Make sure to replace the <version> prefix in the command below with the
-   currently installed MetalK8s version number e.g 2.5.0
+#. From the Bootstrap node, run the following to propagate the
+   changes.
 
-   .. code-block:: shell
+   .. parsed-literal::
 
-      root@bootstrap $ crictl exec -it \
-                         $(crictl ps -q --label io.kubernetes.container.name=salt-master) bash \
-                         -c "salt-run state.sls metalk8s.addons.dex.deployed saltenv=metalk8s-<version>"
+      root@bootstrap $ kubectl exec -n kube-system -c salt-master \\
+                       --kubeconfig /etc/kubernetes/admin.conf \\
+                       salt-master-bootstrap -- bash -c  \\
```

Comment:
Why the `bash -c` here? We shouldn't need it and be able to just run `salt-run ...` directly.